script/gen_requirements_all.py

```diff
@@ -58,19 +58,36 @@
 # will be included in requirements_all_{action}.txt
 
 OVERRIDDEN_REQUIREMENTS_ACTIONS = {
-    "pytest": {"exclude": set(), "include": {"python-gammu"}},
-    "wheels_aarch64": {"exclude": set(), "include": INCLUDED_REQUIREMENTS_WHEELS},
+    "pytest": {"exclude": set(), "include": {"python-gammu"}, "markers": {}},
```

Comment:
Only looking at style (I'm not familiar with what this does), but can we add a comma to ensure it's formatted the same as the ones below?
```suggestion
    "pytest": {
        "exclude": set(),
        "include": {"python-gammu"},
        "markers": {},
    },
```